### DISCUSSION TITLE
Kde: Dark Theme issue

### DISCUSSION THREAD

**Discussion by Spaceboy60** [SpiralLinux/SpiralLinux-project#494] [Q&A]
Don't know if this is debian or spiral related but selecting a dark theme doesn't make things completely dark but a mixture of dark light and unreadable as the picture below shows

![dolphin](https://github.com/user-attachments/assets/c2828674-dd5e-4907-b9e9-409613241484)
![kate](https://github.com/user-attachments/assets/b22d732a-4e05-436c-a245-b9644aa018cf)

**Comment by geckolinux** [comment]
Hi there, please give this a try: https://github.com/orgs/SpiralLinux/discussions/370#discussioncomment-9728061

**Reply by Spaceboy60** [reply]
Ok, didn't have the plasma style or colours to delete.  However choosing that theme fixes the unreadable text and makes everything uniform. However Dolphin is now light rather than dark. Guess it's an issue with the KDE version Debian stable ships with.

**Reply by geckolinux** [reply]
Hi there, it looks like some third party theme component that you installed is causing issues. On a fresh SpiralLinux installation by simply applying the "Breeze Dark" **_global_** theme it correctly changes all the colors of the Plasma interface as well as KDE apps like Dolphin.

![Screenshot from 2024-10-13 07-53-20](https://github.com/user-attachments/assets/6d6baa92-a770-4c9f-86dd-81a937240537)

![Screenshot from 2024-10-13 07-54-11](https://github.com/user-attachments/assets/b0f64a99-24e3-4d3c-afa1-68ff5fbf7171)

**Reply by Spaceboy60** [reply]
> Hi there, please give this a try: https://github.com/orgs/SpiralLinux/discussions/370#discussioncomment-9728061

So only option is to use breeze twilight??  No way to get a completely dark theme working for at least all KDE apps?

**Reply by Spaceboy60** [reply]
> Hi there, it looks like some third party theme component that you installed is causing issues. On a fresh SpiralLinux installation by simply applying the "Breeze Dark" **_global_** theme it correctly changes all the colors of the Plasma interface as well as KDE apps like Dolphin.
> 
> ![Screenshot from 2024-10-13 07-53-20](https://private-user-images.githubusercontent.com/2692138/376036210-6d6baa92-a770-4c9f-86dd-81a937240537.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg4Mzg0MDAsIm5iZiI6MTcyODgzODEwMCwicGF0aCI6Ii8yNjkyMTM4LzM3NjAzNjIxMC02ZDZiYWE5Mi1hNzcwLTRjOWYtODZkZC04MWE5MzcyNDA1MzcucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMTNUMTY0ODIwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OTllN2Q1YTFlMjE2ODI0NmIxYzc1ZWMyZWUzMGQwYmZkZmFiMGNlM2E3YWNjNTRkNzcyMjUzOGFhNTM1NmFlYiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.K_pOdmtAMfIWoO28ZfMNyC3KP9-1GDERYn4VazjM5RI)
> 
> ![Screenshot from 2024-10-13 07-54-11](https://private-user-images.githubusercontent.com/2692138/376036217-b0f64a99-24e3-4d3c-afa1-68ff5fbf7171.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg4Mzg0MDAsIm5iZiI6MTcyODgzODEwMCwicGF0aCI6Ii8yNjkyMTM4LzM3NjAzNjIxNy1iMGY2NGE5OS0yNGUzLTRkM2MtYWZhMS02OGZmNWZiZjcxNzEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMTNUMTY0ODIwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YjFlZDMyZDUzZjFmNmM3Y2U3MDQ5Y2FhMTc1YzBhZmIzMzdhMjdhNDQ3NWRjYmMyMDM5OTI4YWM0M2I1NjhmMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.dlXrgXb7TACWA1t1AMO72GgES0Os1I-HEmkrQBfJPz4)

Pretty sure it was like that from the off, but I'm going to reinstall to be sure.  Maybe I'm missing a component let's see.

**Reply by geckolinux** [reply]
> So only option is to use breeze twilight?? No way to get a completely dark theme working for at least all KDE apps?

No, you can see in my screenshot [above](https://private-user-images.githubusercontent.com/2692138/376036217-b0f64a99-24e3-4d3c-afa1-68ff5fbf7171.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg4NDgyMTYsIm5iZiI6MTcyODg0NzkxNiwicGF0aCI6Ii8yNjkyMTM4LzM3NjAzNjIxNy1iMGY2NGE5OS0yNGUzLTRkM2MtYWZhMS02OGZmNWZiZjcxNzEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMTNUMTkzMTU2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZTIyYzRiYmYwNzY3OGE5OGY2ZjdhYTVhMjdiMDA3OGQ3ZTQ0ODY4ZWZiMTEyZWJlZTlmYzZjZWMyYzllZTdlNCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.3vuD0ZOg-qkPCAO1jq6c8QKVQFIl3OyKN1PTXtfBCxA) that the full "Breeze Dark" global theme also works correctly, and I assume other dark themes should also work.

**Reply by Spaceboy60** [reply]
Reinstalled from the command line, still global theme breeze dark doesn't work on my KDE install. Result is the same as the first post. As is any dark theme.

**Reply by Spaceboy60** [reply]
> > So only option is to use breeze twilight?? No way to get a completely dark theme working for at least all KDE apps?
> 
> No, you can see in my screenshot [above](https://private-user-images.githubusercontent.com/2692138/376036217-b0f64a99-24e3-4d3c-afa1-68ff5fbf7171.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg4NDgyMTYsIm5iZiI6MTcyODg0NzkxNiwicGF0aCI6Ii8yNjkyMTM4LzM3NjAzNjIxNy1iMGY2NGE5OS0yNGUzLTRkM2MtYWZhMS02OGZmNWZiZjcxNzEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMTNUMTkzMTU2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZTIyYzRiYmYwNzY3OGE5OGY2ZjdhYTVhMjdiMDA3OGQ3ZTQ0ODY4ZWZiMTEyZWJlZTlmYzZjZWMyYzllZTdlNCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.3vuD0ZOg-qkPCAO1jq6c8QKVQFIl3OyKN1PTXtfBCxA) that the full "Breeze Dark" global theme also works correctly, and I assume other dark themes should also work.

Ok, wanted to install budgie desktop as well as kde. For some reason, if I install the budgie version and then install kde from the command line (apt install kde-plasma-desktop, apt install kde-standard etc), no dark themes work.

However if I do it the opposite way round, install kde iso, then install budgie from terminal, I have no theme issues and budgie works without issue.
